### PR TITLE
Update records by Knackhq ID and Object ID

### PR DESCRIPTION
**Pull request by michaelachrisco**

### CODE REVIEW

**Comment by michaelachrisco** [comment]
@josephbridgwaterrowe Please review.

File: spec/knackhq/client_spec.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
     subject(:knack_client_request) { client.send(:request) }
     it { is_expected.to be_a Blanket::Wrapper }
     it { is_expected.to respond_to (:get) }
-    # it { is_expected.to respond_to (:post) }
+    it { is_expected.to respond_to (:post) }
```
Comment (josephbridgwaterrowe) [inline spec/knackhq/client_spec.rb:24]: Shouldn't this be put?

Comment (michaelachrisco) [inline spec/knackhq/client_spec.rb:24]: Added in and kept post for future references.

File: lib/knackhq/client.rb [hunk 2]
```diff
@@ -40,22 +43,38 @@ def records(object)
...
+                     .objects(object)
+                     .records
+                     .get(:params => { :rows_per_page => 500 })
+                     .to_h
       return [] if hash_request.empty?
       hash_request.delete(:records)
       [hash_request]
     end
 
+    def update_record(object, knackhq_id, json)
```
Comment (josephbridgwaterrowe) [inline lib/knackhq/client.rb:68]: Weren't we discussing a change in return value, other than an array? I don't see the point in returning an array :snowman:

Comment (michaelachrisco) [inline lib/knackhq/client.rb:68]: How about [:success] or [:fail]? Or just nil...

Comment (josephbridgwaterrowe) [inline lib/knackhq/client.rb:68]: Why would you return an array?

Comment (michaelachrisco) [inline lib/knackhq/client.rb:68]: All the other requests return an array. I don't mind returning the `:success` or `:fail` or on fail, have the application throw an error.

Comment (josephbridgwaterrowe) [inline lib/knackhq/client.rb:68]: Returning an array for something that returns a binary response makes no sense (not in my book anyway) :smile:

I would suggest true or false, or 0 or 1 if there were just two possible results.

**Pull request merged**